src/sb-util.ts

```diff
@@ -39,23 +48,35 @@ The classes below make up the sb-util API.
 const storage = new WeakMap();
 
 export class ScratchProject {
-    public constructor(projectJSON: JSON) {
-        storage.set(this, projectJSON);
+    /**
+     * The project JSON as read from the project.
+     */
+    public projectJSON: SB3ProjectJSON;
+    /**
+     * An interface capable of fetching the assets, later used by AssetCollections.
+     */
+    public assetFetcher: AssetFetcher;
+
+    public constructor(projectJSON: SB3ProjectJSON | JSON, assetFetcher: AssetFetcher = NoAssetFetcher) {
+        this.projectJSON = projectJSON as SB3ProjectJSON;
```

Comment:
Re: convo with @gnarf -- we're moving away from hiding the projectJson in storage. It will now be accessible through ScratchProject